src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1051,6 +1061,52 @@ private void RegisterHomeWorkspace(HomeWorkspaceModel newWorkspace)
 
         #endregion
 
+        #region backup/timer
+
+        /// <summary>
+        /// Backup all the files
+        /// </summary>
+        protected void SaveBackupFiles(object state)
+        {
+            foreach (var workspace in Workspaces)
+            {
+                if (!workspace.HasUnsavedChanges)
+                    continue;
+
+                string fileName;
+                if (string.IsNullOrEmpty(workspace.FileName))
+                {
+                    fileName = "recent_new_file_" + workspace.Guid;
+                    var ext = workspace is HomeWorkspaceModel ? ".DYN" : ".DYF";
+                    fileName += ext;
```

Comment:
As a convention, we typically put strings like `"recent_new_file_"` into `class Configurations` as string constant and refer to it here. Also, extensions are small letters in Dynamo, so `dyn`, `dyf` would be a better choice.